region/brinstar/red.json

```diff
@@ -324,91 +318,103 @@
+                    {"cost": [
+                      {"spikeHits": 2},
+                      [
+                        {"enemyDamage": {
+                          "enemy": 21,
+                          "hits": 1
+                        }}
+                      ]
+                    ]}
+                  ]},
+                  {"and": [
+                    {"canComeInCharged": {
+                      "fromNode": 1,
+                      "framesRemaining": 0
+                    }},
+                    {"cost": [
+                      {"shinesparkFrames": 150}
+                    ]}
+                  ]},
+                  {"and": [
+                    "canIframeSpikeJump",
+                    {"cost": [
+                      {"spikeHits": 6}
+                    ]}
                   ]}
                 ]}
               ],
-              "note": "FIXME There will be varying health reqs here. SpaceJump and IBJ are used separately rather than canFly because of that. The midair springball jump is off a wall, and would also require hjb otherwise.",
               "unlock": null
             }
           ]
         },
         {
-          "from": 3,
+          "from": 2,
           "to": [
             {
               "id": 1,
               "requires": [
                 {"or": [
-                  "canDamageBoostXRayScopeAccess",
+                  {"and": [
+                    "canDamageBoostXRayScopeAccess",
+                    {"cost": [
+                      [
+                        {"enemyDamage": {
+                          "enemy": 21,
+                          "hits": 6
+                        }},
+                        {"enemyDamage": {
+                          "enemy": 29,
+                          "hits": 1
+                        }}
+                      ]
+                    ]}
+                  ]},
                   "SpaceJump",
                   "Grapple",
-                  "canSpringBallJumpMidAir",
                   {"and": [
-                    "HiJump",
-                    "canDash",
-                    "canIframeSpikeJump"
+                    {"canComeInCharged": {
+                      "fromNode": 2,
+                      "framesRemaining": 0
+                    }},
+                    {"cost": [
+                      {"shinesparkFrames": 150}
+                    ]}
                   ]},
                   {"and": [
-                    "canIBJ",
-                    "canIframeSpikeJump"
+                    "canIframeSpikeJump",
+                    {"cost": [
+                      {"spikeHits": 4}
+                    ]},
+                    {"or": [
+                      {"and": [
+                        "HiJump",
+                        "canDash"
+                      ]},
+                      {"and": [
+                        "HiJump",
+                        "canSpringBallJump"
+                      ]},
+                      "canIBJ"
```

Comment:
I think maybe we should give more leniency in spike hits for IBJ. 4 spike hits is with flawless execution, but it's not as easy to setup the ibj as it is to set up the other two strats, and the wavers can get in the way and destroy your attempt, while the reqs don't ask for the PBs you would need to salvage your attempt then.

Reply:
If you use sideway IBJ I think you can get away with 3 spikehits but yes I truly hate setting up IBJ onto spikes.  Adding 2 hits to the IBJ for leniency.